### PR TITLE
Add missing webhook test and improve tests

### PR DESCRIPTION
**Pull request by akrejcir**
**What this PR does / why we need it**:
Added missing test for webhook functionality, and improved test code.

**Release note**:
```release-note
None
```

### CODE REVIEW

**Comment by akrejcir** [comment]
I will add more changes

/hold

**Comment by ksimon1** [comment]
@akrejcir what is the status of this PR?

File: tests/webhook_test.go [hunk 1]
```diff
@@ -164,7 +172,7 @@ var _ = Describe("Validation webhook", func() {
 					foundSsp = getSsp()
 					foundSsp.Spec.TemplateValidator.Placement = &placementAPIValidationValidPlacement
 					return apiClient.Update(ctx, foundSsp, client.DryRunAll)
-				}, twentySecondTimeout, time.Second).ShouldNot(HaveOccurred(), "failed to update SSP CR with valid template-validator placement fields")
```
Comment (ksimon1) [inline tests/webhook_test.go:167]: Why do you remove this variable? Lets define it in env package

Comment (akrejcir) [reply tests/webhook_test.go:167]: I thought the variable was not important. It has no other meaning than 20 seconds, so it can be inlined.

**Pull request merged**